### DISCUSSION TITLE
How can I reference a file bundled with the executable.

### DISCUSSION THREAD

**Discussion by ghost** [pyinstaller/pyinstaller#6515] [Help]
Hello,

I'm currently using PyInstaller to package and distribute this small app I made. As part of this app, I need to use a JSON file that I can write to and read from. This file is where I store a variety of data. In this context, I need to bundle that JSON file with the executable, however, I have met a variety of issues.

Here is the command:
`pyinstaller src/main.py --onefile --add-data "src/dbma/data.json:." --console`

How would I be able to reference the src/dbma/data.json file in my python script?

Thanks.

**Comment by rokm** [comment]
Use `__file__` or `sys._MEIPASS`, as per [docs](https://pyinstaller.readthedocs.io/en/stable/runtime-information.html).
